### DISCUSSION TITLE
"Access to your account has been suspended due to a violation of our Terms of service"

### DISCUSSION THREAD

**Discussion by jeopardization-temp** [community/community#24606] [Apps, API and Webhooks] [answered]
<p>This happened around 2 weeks ago - had a few people telling me they couldn’t access my repository, tried logging into my account and was told my account was suspended for violating GitHub’s terms of service. I checked my email and did not receive any message regarding this. Can someone tell me what’s going on, and if possible, reinstate my account please? I’m happy to oblige with all of GitHub’s policies and it was not my intention to violate them. I’ve made a ticket regarding this issue already, however I haven’t received a response in over a week.<br>
My original account: <a href="https://github.com/jeopardization" rel="noopener nofollow ugc">https://github.com/jeopardization</a><br>
Ticket ID: 1660520<br>
Screenshot:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/1/21dbe134b588b67db32d173b5afb3d0c2121fdf7.png" data-download-href="/uploads/short-url/4PwQus6ANnwcXYYhpDbccWYRCLB.png?dl=1" title="chrome_EEvQ0cbkDD" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/107632868/181085594-c881a030-3230-4c31-97aa-11d09d6c8665.png" alt="chrome_EEvQ0cbkDD" data-base62-sha1="4PwQus6ANnwcXYYhpDbccWYRCLB" width="690" height="343" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21dbe134b588b67db32d173b5afb3d0c2121fdf7_2_690x343.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21dbe134b588b67db32d173b5afb3d0c2121fdf7_2_1035x514.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21dbe134b588b67db32d173b5afb3d0c2121fdf7_2_1380x686.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21dbe134b588b67db32d173b5afb3d0c2121fdf7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">chrome_EEvQ0cbkDD</span><span class="informations">2560×1274 39.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks.</p>

**Comment by queenofcorgis** [comment]
<p>Hi there <a class="mention" href="/jeopardization-temp">@jeopardization-temp</a>,</p>
<p>The GitHub Community Forum team is unable to provide account details nor reinstate your account. Your ticket will be the best avenue for support in this situation, I have checked and it has been routed to the proper team so rest assured it is in the right place. Response times may vary based on resources and the amount of tickets ahead of you. I really appreciate your patience!</p>
<p>Lili :hibiscus:</p>

**Comment by shamirz123** [comment]
My account on this mail [redacted[ got suspended without any reason. i mail github but did'nt get reply my github account suspended without any reason would you help me getting my account back that account is very important for me i urgently need my account back i got the message that access to this acount is got suspended please help

**Comment by b8nomad** [comment]
![image](https://github.com/community/community/assets/91423779/626442d6-0c06-4010-82a6-22c1d36cc735)
mine too!

email : [redacted]

**Comment by queenofcorgis** [comment, marked as answer]
Just a reminder, that the Community is unable to assist with account suspension. You must open a ticket on our [Support page](https://support.github.com/contact) for matters like this and all other account-specific situations. 

Once you open a ticket it will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

Note: I've removed several folks'  private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md). **Please do not share the email of the account in question in this space or any other PII.** 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close and lock this discussion.

Thank you!
